.eslintrc.json

```diff
@@ -45,6 +45,10 @@
     // very noisy with Prettier violations that get autocorrected anyway.
     "prettier/prettier": "off",
 
+    // We use TypeScript props interfaces, which is mostly redundent with prop
+    // types.
+    "react/prop-types": "off",
```

Comment:
Turning this off bc I don't think we need it. See also https://github.com/Recidiviz/covid19-dashboard/pull/3#discussion_r399644074

Reply:
Thank you - I don't normally use them when using typescript, but saw that hey were enabled on eslint.